CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
+- Improved aggrMethod to accept multiple values (#432, partial)

```

Comment:
```suggestion
- Add: aggrMethod accept multiple values separated by comma (#432, partial)
- Add: aggrMethod 'all' to get all the possible aggregations (#432, partial)
```

In addition, documentation at doc/manuals/aggregated-data-retrieval.md (**aggrMethod** bullet) should be modified to explain the new functionality.

Reply:
Resolved in [ee9cded](https://github.com/telefonicaid/fiware-sth-comet/pull/605/commits/ee9cded3b64e3939b8a7935d061e6da400eb4906)